releasenotes/notes/move-integrations-to-internal-langchain-2cd7ff9a1362182d.yaml

```diff
@@ -0,0 +1,10 @@
+---
+deprecations:
+  - |
+    langchain: Deprecates all modules in the ``ddtrace.contrib.langchain`` package. Use attributes exposed in ``ddtrace.contrib.langchain.__all__`` instead.
+  - |
+    logbook: Deprecates the ``ddtrace.contrib.logbook`` module. Use ``ddtrace.contrib.langchain.logbook`` instead.
+  - |
+    logging: Deprecates the ``ddtrace.contrib.logging`` module. Use ``ddtrace.contrib.langchain.logging`` instead.
+  - |
+    loguru: Deprecates the ``ddtrace.contrib.loguru`` module. Use ``ddtrace.contrib.langchain.loguru`` instead.
```

Comment:
nit: Same comment as the other PR. Should we call out that this will officially be removed in the next major?